migrations/versions/4a53667aff6e_add_fielddefinition_and_fieldvalue.py

```diff
@@ -0,0 +1,109 @@
+"""add FieldDefinition and FieldValue
+
+Revision ID: 4a53667aff6e
+Revises: cf2a880aff10
+Create Date: 2025-01-16 23:24:40.316301
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import JSONB
+
+
+# revision identifiers, used by Alembic.
+revision = "4a53667aff6e"
+down_revision = "cf2a880aff10"
+branch_labels = None
+depends_on = None
+
+
+fieldtype_enum = sa.Enum(
+    "TEXT", "MULTILINE_TEXT", "CHOICE_SINGLE", "CHOICE_MULTIPLE", name="fieldtype"
+)
+
+
+def upgrade() -> None:
+    op.create_table(
+        "field_definitions",
+        sa.Column("id", sa.Integer, primary_key=True),
+        sa.Column("username_id", sa.Integer, sa.ForeignKey("usernames.id")),
+        sa.Column("label", sa.String(255), nullable=False),
+        sa.Column(
+            "field_type",
+            fieldtype_enum,
+            default="TEXT",
+        ),
+        sa.Column("required", sa.Boolean, default=False),
+        sa.Column("enabled", sa.Boolean, default=True),
+        sa.Column("encrypted", sa.Boolean, default=False),
+        sa.Column("choices", JSONB, default=[]),
+        sa.Column("sort_order", sa.Integer, default=0),
+    )
+
+    op.create_table(
+        "field_values",
+        sa.Column("id", sa.Integer, primary_key=True),
+        sa.Column(
+            "field_definition_id",
+            sa.Integer,
+            sa.ForeignKey("field_definitions.id", ondelete="CASCADE"),
+        ),
+        sa.Column("message_id", sa.Integer, sa.ForeignKey("messages.id", ondelete="CASCADE")),
+        sa.Column("_value", sa.Text, nullable=False),
+        sa.Column("encrypted", sa.Boolean, default=False),
+    )
+
+    # Create field definitions for every username
+    connection = op.get_bind()
```

Comment:
Need to wrap the connnection in a session like:

https://github.com/scidsg/hushline/commit/eb27915aa35884d2be1b69fc36b523985e17f3a0

Otherwise there will be bugs